trademarks.md

```diff
@@ -0,0 +1,144 @@
+However, if you have any doubts about your intended use of the trademarks, please contact the [Jupyter Trademarks Committee][].
+
+
+### Use of the Jupyter trademark on merchandise
+
+We encourage use of the Jupyter trademarks (logo and word mark) on stickers, hats, mugs, t-shirts and other physical merchandise to promote the project and the broader open-source ecosystem.
+No explicit approval is needed for these uses as long as
+1) the use follows the official Jupyter Brand Guidelines and
+2) the merchandise is provided free of charge.
+We understand that not everyone (including ourselves) can afford to give away merchandise in sufficient quantities to meet demand.
+Because of this we would love help from the community in creating an official place to purchase Jupyter branded
+merchandise with the profits going back to the project through NumFOCUS.
+If you are interested in helping out with this please get in touch with us on the Jupyter Google Group.
+Offering Jupyter branded merchandise for sale without explicit approval is not permitted.
+
+
+## Uses that Always Require Approval
+
+Any commercial use of the Jupyter trademarks in product or company names must be approved first by the Project Jupyter. Some uses, like calling a company "The Jupyter Company," or a product "Jupyter Hosting" or "Jupyter Cloud" will be refused.
+This is because they are overly broad, or confusing as to whether Jupyter is open source or commercial,
+or whether your product or organization is affiliated with or sponsored by Project Jupyter.
+
+Any use of a derived (modified) logo for any commercial or non-commercial purpose must also be approved first by Project Jupyter.
+We will generally be unable to do this, because of the confusion it may cause.
+
+
+## How to Use the Trademarks
+
+Although many uses of Jupyter's trademarks are governed by more specific rules, which appear in the examples below, the following basic guidelines apply to almost any use of Jupyter's trademarks.
+
+0. The Jupyter trademarks are not currently registered.
+(We will post an update to this policy if they are registered later.)
+These marks should be used in accordance with the Jupyter [brand guidelines][],
+and should be accompanied by a symbol for unregistered trademarks: "(TM)" or a small TM "™".
+This may not be removed or obscured and must always be included with the logo.
+This requirement is waived in all contexts where such marks are not normally included: email, online discussion, non-graphical advertisements (when permitted), and academic papers.
+We encourage the use of the symbol whenever possible, but recognize that many non-commercial and informal uses will omit it.
+
+0. If the word "Jupyter" or the Jupyter logos are used in certain contexts, the following statement should accompany its use: "Jupyter" and the Jupyter logos are trademarks or registered trademarks of NumFOCUS, used by ___________ with permission from the Foundation.
+
```

Comment:
I would remove "from the Foundation", either altogether or replace it with "NumFOCUS".  The wording is from the original [Python doc](https://www.python.org/psf/trademarks/) where the sentence reads fine, but here the word "Foundation" kind of dangles.  Since this would be meant to be used later and out of context, I think we should change it.